runtime/gc_base/ContinuationObjectBuffer.hpp

```diff
@@ -45,7 +45,7 @@ class MM_ContinuationObjectBuffer : public MM_BaseVirtual
 	j9object_t _tail; /**< the tail of the linked list of Continuation objects */
 	MM_HeapRegionDescriptor* _region;  /**< the region in which all buffered objects are located */
```

Comment:
please fix the position of '*'

Reply:
Resolved + existing similar issues in all buffer and standard buffer related files. Few `UDATA` spots changed to `uintptr_t`